app-i18n/transifex-client/transifex-client-0.14.2-r1.ebuild

```diff
@@ -31,4 +30,5 @@ src_prepare() {
 	eapply_user
 	sed -i -e 's:test_fetch_timestamp_from_git_tree:_&:' \
 		tests/test_utils.py || die
+	sed -i ':tests_require=["mock>=3.0.5,<4.0"]:d' setup.py || die
```

Comment:
```
>>> Source compiled.
>>> Test phase: app-i18n/transifex-client-0.14.2-r1
 * python3_7: running distutils-r1_run_phase python_test
python3.7 setup.py test --verbose
Warning: 'keywords' should be a list, got type 'tuple'
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
WARNING: The wheel package is not available.
/usr/bin/python3.7: No module named pip
error: Command '['/usr/bin/python3.7', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/tmp/portage/app-i18n/transifex-client-0.14.2-r1/temp/tmpfl1rd2d3', '--quiet', 'mock<4.0,>=3.0.5']' returned non-zero exit status 1.
 * ERROR: app-i18n/transifex-client-0.14.2-r1::testworld failed (test phase):
 *   Tests fail with python3.7
```
```
>: grep -i mock /var/tmp/portage/app-i18n/transifex-client-0.14.2-r1/work/transifex-client-0.14.2/setup.py 
    tests_require=["mock>=3.0.5,<4.0"],
```
I don't know why, but the `:` don't seem to work very well as delimiters. Switch to `/` and it works.

Reply:
Strange, worked fine for me. Replaced anyway.